src/test_servers.py

```diff
@@ -0,0 +1,71 @@
+"""This modulet tests the client and server modules for the CF 401 Python HTTP-Server assignment."""
+
+import pytest
+
```

Comment:
Your tests need to send a message from the client to the server, and test for teh server response.